lib/module.js

```diff
@@ -24,7 +24,8 @@
 const { NativeModule } = require('internal/bootstrap_loaders');
 const util = require('util');
 const { decorateErrorStack } = require('internal/util');
-const { getURLFromFilePath } = require('internal/url');
+const { getURLFromFilePath, getPathFromURL } = require('internal/url');
+const { URL } = require('url');
 const vm = require('vm');
```

Comment:
👆 `URL` is [already exposed](https://github.com/nodejs/node/pull/18281) on the `global` now.
No need to require it.

Reply:
internally we still require it in case it's overridden

Reply:
Ah, good catch, yes.